package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "token.js",
-  "version": "0.4.6",
+  "name": "supertokenai",
+  "version": "0.5.5",
```

Comment:
Hey, thanks for your interest in contributing!

Before this can be merged, you'll need to revert this change so the update will be included in the main`token.js` package (you'll also need to change the name in the changeset files). Feel free to request a review from me whenever this PR is ready.

Reply:
Ha, this is my fault I didn't realize I was trying to merge to your guy's main branch. Sorry about that.